scripts/null_route_helper

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+
+"""
+Utility for blocking and unblocking traffic from given source ip_prefix on ACL tables.
+
+The block operation will insert a DENY rule at the top of the table. The unblock operation
+will remove an existing DENY rule that has been created by the block operation (i.e. it does
+NOT insert an ALLOW rule, only removes DENY rules).
+
+Since SONiC supports multi ACL rules share the same priority, all ACL rules created by null_route_helper will
+use the highest priority(9999).
+
+Example:
+
+Block traffic from 10.2.3.4/32:
+./null_route_helper acl_table_name block 10.2.3.4/32
+
+Unblock all traffic from 10.2.3.4/32:
+./null_route_helper acl_table_name unblock 10.2.3.4/32
+"""
+
+
+from __future__ import print_function
+
+import syslog
+import sys
+import click
+import ipaddress
+
+from swsssdk import ConfigDBConnector
+
+
+CONFIG_DB_ACL_TABLE_TABLE = "ACL_TABLE"
+CONFIG_DB_ACL_RULE_TABLE = "ACL_RULE"
+CONFIG_DB_VLAN_TABLE = "VLAN"
+
+ACTION_ALLOW = "FORWARD"
+ACTION_DENY = "DROP"
+
+# Since SONiC supports multi ACL rules share the same priority, we use 9999 (the highest) for all rules
+ACL_RULE_PRIORITY = 9999
+# The key of rule will be overridden with BLOCK_RULE_ + ip
+ACL_RULE_PREFIX = 'BLOCK_RULE_'
+
+def notice(msg):
+    """
+    Log a NOTICE message to the console and syslog
+    """
+    syslog.syslog(syslog.LOG_NOTICE, msg)
+    print(msg)
+
+
+def error(msg):
+    """
+    Log an ERR message to the console and syslog, and exit the program with an error code
+    """
+    syslog.syslog(syslog.LOG_ERR, msg)
+    print(msg, file=sys.stderr)
+    sys.exit(1)
+
+
+def ip_ver(ip_prefix):
+    return ipaddress.ip_network(ip_prefix, False).version
```

Comment:
>ip_network [](http://example.com/codeflow?start=21&length=10)

If you keep this variable, you can reuse it in future, such as check prefix length.

Reply:
I don't like to add a global variable to save the ip version. Any better ideas?

Reply:
I don't like to add a global variable to save the return value. Any better ideas?

Reply:
I don't like global var either. To clarify, I mean the object of `ip_network` is very useful, and you could deduce many attributes from it such as prefix length, not only the version.